src/parser/view_transform.c

```diff
@@ -444,7 +455,7 @@ mq_bump_corr_pre (PARSER_CONTEXT * parser, PT_NODE * node, void *void_arg, int *
 
   *continue_walk = PT_CONTINUE_WALK;
 
-  if (!PT_IS_QUERY_NODE_TYPE (node->node_type))
+  if (!PT_IS_SELECT (node))
```

Comment:
Is this function not entered for PT_UNION, PT_DIFFERENCE, and PT_INTERSECTION? Or has it been changed to return right here?

Reply:
It is related to commit 093b314
The correlation level refers to the level of the external reference column.
```
select * from tbl a, (select * from tbl2 b 
                                 where b.col1 = a.col1 <== correlated column 
                               ) <== correlation level 2
```
UNION, DIFFERENCE, INTERSECTION is not related to correlation level.
It is bug fix.

Reply:
OK, I see.